client/web/compose/src/components/PageBlocks/RecordBase.vue

```diff
@@ -13,6 +13,11 @@
     <div
       v-else-if="fieldModule"
       class="mt-3 px-3"
+      :class="{
```

Comment:
We talked about this no? Class should be a computed. The common classes shouldn't be repeated.
The computed should just return the value of the key that is the layout mode.